aws/resource_aws_security_group.go

```diff
@@ -1464,8 +1477,11 @@ func networkInterfaceAttachedRefreshFunc(conn *ec2.EC2, id string) resource.Stat
 		}
 		describeResp, err := conn.DescribeNetworkInterfaces(describe_network_interfaces_request)
 
+		if isAWSErr(err, "InvalidNetworkInterfaceID.NotFound", "") {
+			return 42, "false", nil
```

Comment:
What is `42` here?

Reply:
@jen20 something I don't necessarily agree with, but [an existing convention](https://github.com/terraform-providers/terraform-provider-aws/search?q=return+42&unscoped_q=return+42) that is a workaround for `return nil, X, nil` causing `WaitForState()` to treat the refresh as a missing resource and automatically retry without exiting the `WaitForState()` since `Target` has a value: https://github.com/hashicorp/terraform/blob/master/helper/resource/state.go#L115-L137

Reply:
See also: https://en.wikipedia.org/wiki/42_(number)#The_Hitchhiker's_Guide_to_the_Galaxy 😛

Reply:
Heh, the second part was what I assumed but thought I would clarify! Is it worth introducing a constant here to make the intent clearer?